src/SerialPort.cpp

```diff
@@ -414,7 +440,16 @@ namespace CppLinuxSerial {
 
 		//======================== (.c_iflag) ====================//
 
-		tty.c_iflag     &= ~(IXON | IXOFF | IXANY);			// Turn off s/w flow ctrl
+		switch(flowControl_){
```

Comment:
What about FlowControl::HARDWARE in this switch statement? Presumably you'd want to disable software flow control just like in the case of NONE.

Reply:
Both of the two points here follow the same logic, so I'll reply here only.

Reading the codebase I found that the Hardware Flow Control (c_cflag) bit was toggled around line 200, with the Software Control (c_iflag) around line 400 (all in SerialPort.cpp).

While I did think to consolidate them all together, I kept them to two separate switch statements to maintain the structure already in place.

That being said, there would be no issue to group them up if we think that is more appropriate.